plugins/okta/okta_rules.yaml

```diff
@@ -0,0 +1,6 @@
+- rule: Dummy
```

Comment:
Like `falco.yaml` this is probably not the right place for the ruleset. Plus, this seems to be a dummy ruleset used for testing.